portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,83 @@
 <!DOCTYPE html>
+<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
This `<link>` tag should be within the `<head>` tag.  Per [MDN documentation](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML), the `<head>` tag contains information that is not displayed to the user within the webpage itself (e.g. tab title, favicon, any scripts / css that needs to be downloaded).